src/System.Device.Gpio/System/Device/Gpio/PinValue.cs

```diff
@@ -82,6 +82,21 @@ public override bool Equals(object? obj)
         /// </summary>
         public static bool operator !=(PinValue a, PinValue b) => !a.Equals(b);
 
+        /// <summary>
+        /// Returns the inverse of the value
+        /// </summary>
+        /// <param name="v">Input value</param>
+        /// <returns>High, when the input is low, low when the input is high</returns>
+        public static PinValue operator !(PinValue v)
+        {
+            if (v._value == 0)
+            {
+                return High;
+            }
+
+            return Low;
+        }
```

Comment:
```suggestion
        public static PinValue operator !(PinValue v) => (v._value == 0) ? High : Low;
```

Reply:
Personally, I prefer the long form (especially, if it includes logic, like here). Are there any recommendations on this?